### ISSUE TITLE
ddl-resource.sql has a bug

### ISSUE THREAD

**Issue by rh-hemartin** [kubearchive/kubearchive#384]
When I try to execute this as production:

```
systemd-run -p Delegate=yes --user --scope kind create cluster
bash hack/install-dependencies.sh
# delete the last line of hack/install-database.sh
bash hack/install-database.sh
wget https://github.com/kubearchive/kubearchive/releases/latest/download/ddl-resource.sql
kubectl port-forward -n databases svc/postgresql 5432:5432 &
psql -h localhost -U root -f ddl-resource.sql
```

Outputs

```
[hemartin@hemartin-thinkpadp1gen3 kubearchive]$ psql -h localhost -U root -f ddl-resource.sql
Handling connection for 5432
Handling connection for 5432
CREATE SCHEMA
CREATE TABLE
CREATE FUNCTION
psql:ddl-resource.sql:35: ERROR:  function trigger_set_timestamp() does not exist
ALTER TABLE
GRANT
ALTER FUNCTION
GRANT
GRANT
```

**Comment by rh-hemartin** [comment]
Nevermind, `ddl-resource.sql` should be executed as the user created, not as the admin user.

**Comment by maruiz93** [comment]
I think that the DDL should be able to be run by whatever user, IINM the bug here is in [this line](https://github.com/maruiz93/kubearchive/blob/main/database/ddl-resource.sql#L35), should be:

`kubearchive.resource for each row execute function kubearchive.trigger_set_timestamp();`

instead of 

`kubearchive.resource for each row execute function trigger_set_timestamp();`

I will reopen the issue

**Comment by mafh314** [comment]
> I think that the DDL should be able to be run by whatever user with enough permissions
Hey @maruiz93  I'm not about this. In https://github.com/kubearchive/kubearchive/issues/271#issue-2434216380 we have: "Create a kubearchive user who is the owner of the kubearchive database. This user should be used for all KubeArchive database connections."

**Comment by maruiz93** [comment]
> > I think that the DDL should be able to be run by whatever user with enough permissions
> 
> Hey @maruiz93 I'm not about this. In [#271 (comment)](https://github.com/kubearchive/kubearchive/issues/271#issue-2434216380) we have: "Create a kubearchive user who is the owner of the kubearchive database. This user should be used for all KubeArchive database connections."

With "This user should be used for all KubeArchive database connections." I understand that the Kubearchive DB user should be used by the sink and the KubeArchive REST API to interact with the database, but for running the DDL, as it is something that should be done by a DB administrator of the DB that we don't own, it's ok if they want to apply them by a `root` or other privileged user. But that doesn't mean that the DDL couldn't be applied by the kubearchive user.

In other words, I agree with that sentence, but I don't think that means that the DDL could be executed by any other privileged user. Applying the DDL is part of the installation process and not a part of the KubeArchive usual workflow.

Also, there is no way to prevent a privileged database user of being able to execute DDL statements over kubearchive database.
